hubspot/plugin.py

```diff
@@ -335,59 +876,40 @@ def list_companies(
 
     companies = []
     for item in data["results"]:
-        property_values = item.get("properties", {})
-        additional_property_values = {}
-        if additional_properties:
-            for addl in additional_properties:
-                val = property_values.get(addl, None)
-                if val:
-                    additional_property_values[addl] = val
-
-        company_properties = HubSpotCompanyProperties(
-            name=property_values["name"],
-            domain=property_values["domain"],
-            hs_object_id=item["id"],
-            last_modified_date=datetime.fromisoformat(
-                property_values.get("lastmodifieddate", "1970-01-01T00:00:00Z")
-            ),
-            additional_properties=additional_property_values,
-        )
-        company = HubSpotCompany(
-            id=item["id"],
-            properties=company_properties,
-            createdAt=datetime.fromisoformat(item["createdAt"]),
-            updatedAt=datetime.fromisoformat(item["updatedAt"]),
-            archived=item["archived"],
-        )
+        company = _parse_company(item)
         companies.append(company)
 
-    next_after = (
+    next_pagination_token = (
         data["paging"]["next"]["after"]
         if "paging" in data and "next" in data["paging"]
         else None
     )
 
-    return companies, next_after
+    return companies, next_pagination_token
 
 
-def search_companies(
+def hubspot_search_companies(
     search_criteria: Dict[str, str],
```

Comment:
should the first elem in this Dict  be a type like HubSpotCompanyDefaultPropertyName ?

Reply:
(using Seq[Tuple[...]] ok )

Reply:
I will follow this up in the next PR.